lib/citizens_advice_form_builder/elements/collections/radio_buttons.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module CitizensAdviceFormBuilder
+  module Elements
+    module Collections
+      class RadioButtons < Base
+        include ActionView::Helpers::FormOptionsHelper
+
+        def render
+          component = CitizensAdviceComponents::RadioGroup.new(
+            legend: label,
+            name: field_id,
```

Comment:
A big aside, I think the recent design system v5.7.0 will impact how the name/id works once it's rolled out